sequencer/Cargo.toml

```diff
@@ -58,7 +58,7 @@ derive_more = { workspace = true }
 dotenvy = { workspace = true }
 either = { workspace = true }
 espresso-contract-deployer = { path = "../contracts/rust/deployer" }
-espresso-types = { path = "../types" }
+espresso-types = { path = "../types", features = ["testing"] }
```

Comment:
Why do we need this? The testing feature is enabled for the test build already. It would be best to not enable it for the non-test build.